src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -390,6 +415,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // Method Description:
     // - Search text in text buffer. This is triggered if the user click
     //   search button or press enter.
+    // In the live search mode it will be also triggered once every time search criteria changes

```

Comment:
You know, this comment suggests the existence of a non-live search mode. Have we done away with that? Should we update the comments to be more strictly correct?